src/components/Profile/Profile.js

```diff
@@ -1,16 +1,25 @@
+import { useContext } from 'react';
+import ContentViewContext from 'contexts/ContentViewContext';
 import 'components/Profile/Profile.scss';
 import profile from 'assets/profile.png';
```

Comment:
it's a good practice to leave some empty lines to split imports
```suggestion
import { useContext } from 'react';

import ContentViewContext from 'contexts/ContentViewContext';

import 'components/Profile/Profile.scss';
import profile from 'assets/profile.png';
```

Reply:
Done!